stdlib/caffe/caffe_input_kernel.cpp

```diff
@@ -156,10 +156,12 @@ void CaffeInputKernel::transform_caffe(u8 *input_buffer, u8 *output_buffer) {
 
 void CaffeInputKernel::execute(const BatchedColumns &input_columns,
                                BatchedColumns &output_columns) {
-  check_frame_info(device_, input_columns[1]);
+  auto& frame_col = input_columns[1];
+  auto& frame_info_col = input_columns[2];
+  check_frame_info(device_, frame_info_col);
```

Comment:
I don't believe you need to avoid the `index` column here because the kernel only requests two inputs: frame and frame_info. Those should still come in as the 0th and 1st column.